### PR TITLE
Test for: Check null should return

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-devpack-dotnet/issues/527

### CODE REVIEW

File: tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_NULL.cs [hunk 1]
```diff
@@ -42,6 +42,11 @@ public static string NullCollation(string code)
             return myname;
         }
 
+        public static bool NullProperty(string a)
+        {
+            return a?.Length != 0;
```
Comment (cloud8little) [inline tests/Neo.Compiler.CSharp.UnitTests/TestClasses/Contract_NULL.cs:47]: not != , but >, or < , will leads to error.

**Comment by erikzhang** [comment]
@cloud8little Please check for neo-project/neo-vm#405

**Comment by cloud8little** [comment]
Let me add several UT cases.

**Review by cloud8little — APPROVED** [review]

**Comment by cloud8little** [comment]
Good to merge.

**Pull request merged**